Airports/EGCN/VRPs.txt

```diff
@@ -1,10 +1,8 @@
 *Goole Docks:N053.41.49.000:W000.52.40.000
 *Stainforth M18:N053.35.29.000:W000.59.15.000
-*Thorpe Marsh:N053.34.53.000:W001.05.20.000
 *Haxey:N053.29.25.000:W000.50.37.000
 *Wadworth M18/A1M:N053.28.45.000:W001.08.54.000
-*Gainsborough:N053.23.34.000:W000.46.15.000
```

Comment:
Put this line back

Reply:
Have done in my latest bit, will add to Changelog.